src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -206,7 +206,7 @@ public async Task ExecuteAsync(ExportJobRecord exportJobRecord, WeakETag weakETa
             }
             catch (DestinationConnectionException dce)
             {
-                _logger.LogError(dce, "Can't connect to destination. The job will be marked as failed.");
+                _logger.LogInformation(dce, "Can't connect to destination. The job will be marked as failed.");
```

Comment:
Should we log it as a warning?
If export fails, how are we doing to detect it from IcMs?

Reply:
Then it's fine.